### ISSUE TITLE
Error: using `ember-decorators` blueprints instead of the `ember` blueprints.

### ISSUE THREAD

**Issue by ppcano** [ember-cli/ember-octane-blueprint#37]
The issue was previously reported at [Discord](https://discordapp.com/channels/480462759797063690/484527343331704832/545909591800807426). I am not sure if this is the correct repo to post this info.

Executing `ember g controller foo-bar` will generate the [`ember-decorator` blueprints](https://github.com/ember-decorators/ember-decorators/tree/master/packages/-ember-decorators/blueprints) instead of the [`ember` blueprints](https://github.com/emberjs/ember.js/tree/master/blueprints).

The `ember-decorators` blueprints don't include the latest MU fixes and they failed in some cases.

```
installing controller
  create app/controllers/foo-bar.js
installing controller-test
  create src/ui/routes/foo-bar/controller-test.js
```

At this moment, I am working on updating the Ember blueprint to use native classes at https://github.com/emberjs/ember.js/pull/17621

@pzuraq what should it be the strategy to fix the issue?

A possible proposal:

- Complete my PR and get merged.

- Remove the `ember-decorators` blueprints.

- Update the octane blueprint to use the new releases.

**Comment by rwjblue** [comment]
We should:

* remove `ember-decorators` from this repo
* enable the `EMBER_NATIVE_DECORATOR_SUPPORT` feature flag
* enable the `EMBER_METAL_TRACKED_PROPERTIES` feature flag

**Comment by NullVoxPopuli** [comment]
I need to do some testing on this tonight. Been having issues with emberclear and native decorators

**Comment by NullVoxPopuli** [comment]
for those upgrading, we'll probably want to track whatever ends up being the solution to this:
![image](https://user-images.githubusercontent.com/199018/52890421-9fd31180-3152-11e9-869c-1de96d446ce9.png)
(though, I still have ember-decorators)

**Comment by ppcano** [comment]
@rwjblue Is this a change we can do at this moment, or do we have to wait for more progress on `EMBER_NATIVE_DECORATOR_SUPPORT` and `EMBER_METAL_TRACKED_PROPERTIES`?

**Comment by rwjblue** [comment]
I think we can do it now (at least enabling the two other flags). We _might_ run into problems, but that’s why it’s important to test things...

**Comment by rwjblue** [comment]
Oh, also, we should add @glimmer/components addon (the most recent alpha release)

**Comment by NullVoxPopuli** [comment]
resolved by v0.4.0
